csrankings-2.csv

```diff
@@ -1610,7 +1610,7 @@ Florent Capelli,CRIStAL,https://www.cristal.univ-lille.fr/profil/fcapelli,NOSCHO
 Florentin Wörgötter,University of Göttingen,http://www.uni-goettingen.de/en/58059.html,NOSCHOLARPAGE
 Florian Alt,Bundeswehr University Munich,https://www.unibw.de/usable-security-and-privacy/team/univ-prof-dr-florian-alt,wf3Muy8AAAAJ
 Florian Daniel,Politecnico di Milano,http://home.deib.polimi.it/daniel,wF6n9-0AAAAJ
-Florian Echtler,Bauhaus University Weimar,https://www.uni-weimar.de/de/medien/professuren/medieninformatik/mobile-media/people,sZssYagAAAAJ
+Florian Echtler,Aalborg University,https://floe.butterbrot.org/,sZssYagAAAAJ

```

Comment:
Not listed here: https://vbn.aau.dk/da/organisations/institut-for-datalogi/persons/

Reply:
Listed here: https://vbn.aau.dk/en/persons/149493